plugins/modules/helm_cli_info.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: helm_cli_info
+short_description: Get informations from Helm package deployed inside the cluster
+description:
+  -  Get informations (values, states, ...) from Helm package deployed inside the cluster
+version_added: "2.10"
+author:
+  - Lucas Boisserie (@LucasBoisserie)
+requirements:
+  - "helm (https://github.com/helm/helm/releases)"
+  - "yaml (https://pypi.org/project/PyYAML/)"
+options:
+  binary_path:
+    description:
+      - The path of a helm binary to use.
+    required: false
+    type: path
+  release_name:
+    description:
+      - Release name to manage
+    required: true
+    type: str
+    aliases: [ name ]
+  release_namespace:
+    description:
+      - Kubernetes namespace where the chart should be installed
+      - Can't be changed with helm 2
+    default: "default"
+    required: false
+    type: str
+    aliases: [ namespace ]
+  tiller_host:
+    description:
+      - Address of Tiller
+      - Ignored when is helm 3
+    type: str
+  tiller_namespace:
+    description:
+      - Namespace of Tiller
+      - Ignored when is helm 3
+    default: "kube-system"
+    type: str
+
+#Helm options
```

Comment:
For the auth options, do you think you could make use of the shared [`AUTH_ARG_SPEC`](https://github.com/ansible-collections/kubernetes/blob/master/plugins/module_utils/common.py#L97) that the other modules use?

Reply:
The shared spec offer too many options that helm doesn't manage
Helm has only 2 options to configure k8s auth (kube-context && kubeconfig)

Reply:
ah that makes sense. How does it behave if you are running the helm commands from inside a cluster? I assume it will make use of the incluster config, but perhaps that's just automatic?

Reply:
I just test inside a pod and it used the SA. Nothing is explained inside the helm documentation so I check helm's source code. 
It seems to load the incluster config with `k8s.io/cli-runtime` package from env vars.